# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_statement_import_online_ponto
#
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 15.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-08 00:29\n"
"PO-Revision-Date: 2024-11-08 00:29\n"
"Last-Translator: zvERP <galego@zverp.com>\n"
"Language-Team: Xacobe\n"
"Language: gl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: tradutor Zerever\n"

#. module: account_statement_import_online_ponto
#: code:addons/account_statement_import_online_ponto/models/online_bank_statement_provider_ponto.py:0
#, python-format
msgid ""
"Error during Create Synchronization %(code)s \n"
"\n"
" %(text)s"
msgstr ""
"Erro durante a sincronización de creación %(code)s \n"
"\n"
" %(text)s"

#. module: account_statement_import_online_ponto
#: code:addons/account_statement_import_online_ponto/models/online_bank_statement_provider_ponto.py:0
#, python-format
msgid ""
"Error during get transaction.\n"
"\n"
"%(code)s\n"
"\n"
"%(text)s"
msgstr ""
"Erro durante a obtención da transacción.\n"
"\n"
"%(code)s\n"
"\n"
"%(text)s"

#. module: account_statement_import_online_ponto
#: model_terms:ir.ui.view,arch_db:account_statement_import_online_ponto.online_bank_statement_provider_form
msgid "Login"
msgstr "Iniciar sesión"

#. module: account_statement_import_online_ponto
#: model:ir.model,name:account_statement_import_online_ponto.model_online_bank_statement_provider
msgid "Online Bank Statement Provider"
msgstr "Provedor de Extractos Bancarios en Liña"

#. module: account_statement_import_online_ponto
#: code:addons/account_statement_import_online_ponto/models/online_bank_statement_provider_ponto.py:0
#, python-format
msgid "Please fill login and key."
msgstr "Por favor, encher login e clave."

#. module: account_statement_import_online_ponto
#: code:addons/account_statement_import_online_ponto/models/online_bank_statement_provider_ponto.py:0
#, python-format
msgid "Ponto : no token"
msgstr "Ponto: sen ficha"

#. module: account_statement_import_online_ponto
#: code:addons/account_statement_import_online_ponto/models/online_bank_statement_provider_ponto.py:0
#, python-format
msgid "Ponto : wrong configuration, unknow account %s"
msgstr "Ponto : configuración errónea, conta descoñecida %s"

#. module: account_statement_import_online_ponto
#: model:ir.model.fields,field_description:account_statement_import_online_ponto.field_online_bank_statement_provider__ponto_last_identifier
msgid "Ponto Last Identifier"
msgstr "Ponto Último Identificador"

#. module: account_statement_import_online_ponto
#: model:ir.model.fields,field_description:account_statement_import_online_ponto.field_online_bank_statement_provider__ponto_token
msgid "Ponto Token"
msgstr "Ficha Ponto"

#. module: account_statement_import_online_ponto
#: model:ir.model.fields,field_description:account_statement_import_online_ponto.field_online_bank_statement_provider__ponto_token_expiration
msgid "Ponto Token Expiration"
msgstr "Caducidade das fichas Ponto"

#. module: account_statement_import_online_ponto
#: model_terms:ir.ui.view,arch_db:account_statement_import_online_ponto.online_bank_statement_provider_form
msgid "Reset Last identifier."
msgstr "Restablecer Último identificador."

#. module: account_statement_import_online_ponto
#: model_terms:ir.ui.view,arch_db:account_statement_import_online_ponto.online_bank_statement_provider_form
msgid "Secret Key"
msgstr "Clave Secreta"

#, python-format
#~ msgid ""
#~ "Error during Create Synchronisation {} \n"
#~ "\n"
#~ " {}"
#~ msgstr ""
#~ "Erro durante a Sincronización de Creación {} \n"
#~ "\n"
#~ " {}"

#, python-format
#~ msgid ""
#~ "Error during get transaction.\n"
#~ "\n"
#~ "{} \n"
#~ "\n"
#~ " {}"
#~ msgstr ""
#~ "Erro durante a obtención da transacción.\n"
#~ "\n"
#~ "{}\n"
#~ "\n"
#~ " {}"
